Puppet/Puppet/MSMainViewController.m

```diff
@@ -70,4 +72,30 @@ - (IBAction)enableOneCollectorSwitchUpdated:(UISwitch *)sender {
   [self presentViewController:alert animated:YES completion:nil];
 }
 
+- (IBAction)selectTarget:(UISegmentedControl *)sender {
```

Comment:
Question for reviewers: I copied this from another method but this formatting looks atrocious to me.  Is this how it's supposed to be in this case?  Line up all the `:`?